doc/man1/openssl.pod

```diff
@@ -545,6 +545,22 @@ Read the password from standard input.
 
 =back
 
+=head2 Engine Options
+
+=over
+
+=item B<-engine> I<id>
+
+Specifying an engine with the indicated B<id> will cause the command
+to attempt to obtain a functional reference to the specified engine,
+initialiasing it if needed.
```

Comment:
This goes into details that is incomprehensible to those who don't know the ins and outs of the ENGINE API. That's all of those that just use the command line.

```
Use the engine identified with I<id> and use all the methods it implements (algorithms, key storage,...), unless
```

and then continue with what you have after "unless" below

Reply:
Thanks.  That text (as you know, since your #10065 touches on it), was pulled from existing pages.  I agree with your comment 1000%  Took your wording.